src/components/admin/applications/ApplicationDetailPage.tsx

```diff
@@ -60,6 +121,7 @@ const ApplicationDetailPage: React.FC = () => {
               }}
             />
           </Tag>
+          <Button onClick={onOpen} size="xs" colorScheme='messenger'>Applicant Settings</Button>
```

Comment:
Can you change this button to "Change Applicant Settings" and make it a little bigger and more prominent?